test/google_crawler_web/controllers/api/keyword_controller_test.exs

```diff
@@ -34,4 +34,78 @@ defmodule GoogleCrawlerWeb.Api.KeywordControllerTest do
              } = json_response(conn, 200)
     end
   end
+
+  describe "show/2" do
+    test "returns the keyword with search results for the given keyword id", %{conn: conn} do
+      user = insert(:user)
+      %{title: keyword_title, status: keyword_status} = keyword = insert(:keyword, user: user)
+      %{title: search_result1_title} = insert(:search_result, keyword: keyword)
+      %{title: search_result2_title} = insert(:search_result, keyword: keyword)
+
+      conn =
+        conn
+        |> login_as(user)
+        |> get(Routes.keyword_path(conn, :show, keyword.id))
+
+      assert %{
+               "data" => %{
+                 "attributes" => %{
+                   "status" => ^keyword_status,
+                   "title" => ^keyword_title
+                 },
+                 "type" => "keyword",
+                 "relationships" => %{
+                   "search_results" => %{
+                     "data" => [
+                       %{
+                         "type" => "search_result"
```

Comment:
I'm just curious if this is ordered in any particular way? Is it by inserted in ascending order? 🤔

Reply:
the order does not specify in preload so the order comes out in database default order. it should be `id`